.haml-lint.yml

```diff
@@ -0,0 +1,13 @@
+linters:
+  LineLength:
+    enabled: false
+
+  ClassesBeforeIds:
+    EnforcedStyle: 'id'
+
+  SpaceInsideHashAttributes:
+    enabled: false
+
+  RuboCop:
+    # @TODO: Probably turn this back on
+    enabled: false
```

Comment:
This is also disabled in public-website. Enabling triggers a bunch of small warnings, worth fixing eventually but not work blocking a build for.

Reply:
Question) If this is being ran as a separate script. Does this need to be present here? Or will it auto-run it without it being present?

Reply:
It auto-runs as part of haml-lint